### PR TITLE
Feature/slider

### PR DESCRIPTION
**Pull request by madvingovich**
SliderWrapper - that's how we gonna use it (I don't know how to put childred to component from storybook)
Only for mobiles for now
slider action creates custom events for us

### CODE REVIEW

File: src/actions/slider.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  function handleMousedown(e) {
+    x = typeof e.clientX == 'number' ? e.clientX : e.touches[0].clientX;
+    node.dispatchEvent(
+      new CustomEvent('slideStart', {
+        detail: { x }
+      })
+    );
+
+    window.addEventListener('mousemove', handleMousemove);
```
Comment (dimkabuki) [inline src/actions/slider.js:12]: Why window and not document.body?

File: src/stories/datepicker/datepicker.stories.js [hunk 2]
```diff
@@ -7,6 +7,8 @@ const change = (selectedDates, dateStr, instance) => {
   birthDate.set(dateStr);
 };
 
+const additionalClass = 'text-center mx-auto mt-5 shadow-lg';
```
Comment (dimkabuki) [inline src/stories/datepicker/datepicker.stories.js:10]: Nice abstraction! ;)

File: src/actions/slider.js (hunk 1 above)
Comment (madvingovich) [reply src/actions/slider.js:12]: Actually examples that I looked used window) I never thought about difference.

Comment (madvingovich) [reply src/actions/slider.js:12]: As I see, the only difference in my case is that document.body is closer to the source

Comment (dimkabuki) [reply src/actions/slider.js]: That's my point.
Using global object is tricky.
Imagin you will have to add another slider effect in the app. Where would you place the handler?

**Pull request merged**